lcUILua/createActions/circleoperations.lua

```diff
@@ -12,19 +15,36 @@ setmetatable(CircleOperations, {
 
 function CircleOperations:_init(id)
     CreateOperations._init(self, id, lc.builder.CircleBuilder, "enterCenter")
-
+    cli_get_text(self.target_widget, true)
```

Comment:
This commands prevents the user from entering a coordinate in the command line. 
There isn't a proper way to handle this right now, but you could add a comment explaining that this is the cause of the problem.